docker-compose.yml

```diff
@@ -372,6 +372,7 @@ services:
       - mariadb
     environment:
       - _APP_ENV
+      - _APP_CONNECTIONS_DB_QUEUE_01
```

Comment:
Do we need the number postfix? Would be more elegant if we can skip on the default self-hosted setup.